config/routes.rb

```diff
@@ -12,5 +12,12 @@
     end
   end
   
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
   get '/about', to: 'about#about'
+
```

Comment:
[Optional] Typing the command `rails routes | grep user` should yield URLs that correspond to these wireframes
It is much better to write it this way: 
 ```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

Reply:
okay